response_operations_ui/views/errors.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+from flask import Blueprint, render_template
+from structlog import wrap_logger
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+error_bp = Blueprint('error_bp', __name__, template_folder='templates/errors')
+
+
+@error_bp.route('/500', methods=['GET', 'POST'])
+def server_error_page():
+    return render_template('errors/500-error.html', _theme='default',
+                           data={"error": {"type": "failed"}}), 500
```

Comment:
Does this `data` variable do anything here?

Reply:
It does nothing, removed